src/raftstore/store/snap.rs

```diff
@@ -852,10 +856,21 @@ impl Snapshot for Snap {
             if plain_file_used(cf_file.cf) {
                 let path = cf_file.path.to_str().unwrap();
                 let batch_size = options.write_batch_size;
-                snap_io::apply_plain_cf_file(path, &abort_checker, &options.db, cf, batch_size)?;
+                // let cb = |key| {
```

Comment:
please remove commented out code

Reply:
Sorry. This is a mistake. I'll update this PR later and the implementation will be slightly changed.